Sources/Purchasing/StoreKitAbstractions/StoreTransaction.swift

```diff
@@ -112,6 +113,10 @@ internal protocol StoreTransactionType: Sendable {
     /// - Note: this is only available for StoreKit 2 transactions starting with iOS 17.
     var storefront: Storefront? { get }
 
+    /// The raw JWS repesentation of the transaction.
+    /// - Note: this is only available for StoreKit 2 transactions.
+    var jwsRepresentation: String? { get }
```

Comment:
I was thinking about this, let's make this `internal` that way we don't have to make this PR require a minor release?